template/testsite_builder.config_template.views_node_content.yml

```diff
@@ -0,0 +1,124 @@
+template:
+  entity_type: 'node'
+  source: 'views.view.content'
+  pre_generate_clone:
+    -
+      from: 'display.default.display_options.fields.node_bulk_form'
+      to: 'display.default.display_options.fields.node_bulk_form'
+  post_generate_clone:
+    -
+      from: 'display.default.display_options.fields.operations'
+      to: 'display.default.display_options.fields.operations'
+  generate_per_field:
+    -
+      type: 'view_dependency_config'
+      name: 'dependencies.config'
+    -
+      type: 'view_style_column'
+      name: 'display.default.display_options.style.options.columns'
+    -
+      type: 'view_style_info'
+      name: 'display.default.display_options.style.options.info'
+    -
+      type: 'view_field'
+      name: 'display.default.display_options.fields'
+    -
+      type: 'view_filter'
+      name: 'display.default.display_options.filters'
+    -
+      type: 'view_config_cache_tag'
+      name: 'display.default.cache_metadata.tags'
+    -
+      type: 'view_config_cache_tag'
+      name: 'display.page_1.cache_metadata.tags'
+  generate_per_view:
+    -
+      type: 'view_bundle_path'
+      name: 'display.page_1.display_options.path'
+  field_type_mapping:
+    - type: 'entity_reference'
+      match_rules:
+        storage_settings:
+          target_type: 'taxonomy_term'
+      source_field: 'field_channel'
+      custom_generation_types:
+        -
+          type: 'view_filter'
```

Comment:
Lets ghet rid of these inline config, and replace it with the fallback configuration files, that we talked about